File: fintechplatform/src/main/java/com/dwaplatform/android/alert/AlertHelpersModule.java
package com.dwaplatform.android.alert;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;

/**
 * Created by ingrid on 08/12/17.
 */

@Module
public class AlertHelpersModule {

    @Provides
    @Singleton
    public AlertHelpers provideAlertHelpers() {
        return new AlertHelpers();
    }
}

File: dwaplatform/src/main/java/com/dwaplatform/android/alert/AlertHelpersModule.kt
package com.dwaplatform.android.alert

/**
 * Created by tcappellari on 08/12/2017.
 */

/**
 * Created by ingrid on 07/09/17.
 */

@Module
class AlertHelpersModule {

    @Provides
    @Singleton
    internal fun provideAlertHelpers(): AlertHelpers {
        return AlertHelpers()
    }
}

